dashboard/Makefile

```diff
@@ -0,0 +1,24 @@
+GOCMD ?= go
+GOARGS= -trimpath
+REPLICATOR_OUTPUT_DIR ?= bin
+
+replicator: fmt vet
+	$(GOCMD) build $(GOARGS) -o $(REPLICATOR_OUTPUT_DIR)/replicator cmd/postgres_replicator/main.go
+
+configure_replicator:
+	scripts/check_env.sh GCP_PROJECT_ID GCP_DATA_TRANSFER_SERVICE BQ_PROJECT_ID PG_USER PG_PASS PG_DATABASE CLOUD_SQL_INSTANCE
+	cp ../config/dashboard/postgres_replicator/config.yaml postgres_replicator
```

Comment:
I don't think you should copy and then substitute in place. It would make more sense to have a generic "envsubst.sh"  that takes one input and one output and performs the substitutions. This can perhaps be done in a follow-up.

Reply:
Done. Now I just use plain old `envsubst` in the make file, taking the files from `dasboard/config` and placing them in a new `containers` directory